modules/mod_authentication/controllers/controller_logon.erl

```diff
@@ -690,7 +690,14 @@ delete_reminder_secret(Id, Context) ->
 
 
 get_by_reminder_secret(Code, Context) ->
-    case m_identity:lookup_by_type_and_key("logon_reminder_secret", Code, Context) of
+    Result = case m_config:get_value(mod_authentication, reset_token_maxage, Context) of
+        undefined ->
+            m_identity:lookup_by_type_and_key("logon_reminder_secret", Code, Context);
+        MaxAge ->
+            m_identity:lookup_by_type_and_key("logon_reminder_secret", Code, z_convert:to_integer(MaxAge), Context)
```

Comment:
Should `MaxAge` also be shown in `email_password_reset.tpl`?

Reply:
Not needed in the email, we should just mention that it can be used for a short period of time.